Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```
@@ -4410,6 +4423,90 @@ script:
     description: Creates a WildFire analysis best practice profile.
     execution: false
     name: panorama-create-wildfire-best-practice-profile
+  - arguments:
+    - default: false
+      description: The template to use when running the command. Overrides the template
+        parameter (Panorama instances). If not given, will use the integration paramater.
+      isArray: false
+      name: template
+      required: false
+      secret: false
+    - default: false
+      description: The template stack to use when running the command.
+      isArray: false
+      name: template_stack
+      required: false
+      secret: false
+    - default: false
+      defaultValue: vsys1
+      description: The name of the virtual system to be configured. Will use the configured
+        Vsys if exists.
```

to your consideration
```suggestion
      description: The name of the virtual system to be configured. Will use the configured
        VSYS paramater if exists, if given a value, will override the VSYS parametmer.
```

---

updated a better description